blog/2023-06-14-paralus-at-platformcon-2023/index.md

```diff
@@ -0,0 +1,114 @@
+
+This event brought together a global community of techies, developers, and platform engineers enthusiasts to share the latest advancements and trends in the field of platform engineering. 
+
+The event also featured a diverse range of talks, covering topics ranging from platform-culture, platform-blueprint, platform-impact to platform-stories. While it was impossible to attend all of them, we have been revisiting some of the talks that were shared on [Platform Engineering YouTube Channel](https://www.youtube.com/@PlatformEngineering).  
+
+<!--truncate -->
+
+In this blog post, we'll be sharing our experience of interacting with attendees at [PlatformCon 2023](https://platformcon.com/) virtually, some glimpses of our talk [Enable secure self-service access to Kubernetes clusters with Paralus](https://www.youtube.com/watch?v=UbLnCGOv0Rk). how paralus removes the cognitive load from platform team's working on repetitive tasks including setting up RBAC, revoking permissions, pre-configuring roles across clusters, namespaces, projects, and more.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/UbLnCGOv0Rk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+During [PlatformCon 2023](https://platformcon.com/), we were inundated with visitors reached us on Platform Engineering community [slack channel](https://platformengin-b0m7058.slack.com/join/shared_invite/zt-1vsqjlb51-SiDy9cp2EBGffIUj5_g2mA#/shared-invite/email), who were keen to learn more about our project. We had some quick demos on how to use the project, along with some tasty cookies and cool stickers to give away. The visitors were curious about the project and its potential, and some of them even reached out for a dedicated demo of the tool post-platformcon.   
+
+Big topic during PlatformCon was the concept of platform engineering. As software gets more complex and Kubernetes enters the enterprise, the idea of Kubernetes being a “platform for platforms” is becoming more and more ubiquitous. 
+
+A bit about “Platform engineering” is a discipline focused on building and maintaining the underlying infrastructure and tools that enable developers to build, deploy, and run software applications efficiently. It involves creating a stable and scalable platform that provides a foundation for software development, deployment, and operations.
+
+**[Six Guiding Principles For Platform Teams](https://www.forbes.com/sites/forbestechcouncil/2023/02/23/how-to-empower-modern-kubernetes-management-with-a-platform-team-model/?sh=1690484c52c7)**: 
+- Consistent Automation
```

Comment:
Let us add a one-liner description for each of these principles.

Reply:
Also let us make these as bullet points as they are easy to read and comprehend.